Makefile

```diff
@@ -103,7 +103,7 @@ gen-themes:
 ## Generate Icons
 .PHONY: gen-icons
 gen-icons:
-	@cd $(ICONS_DIR) && $(PNPM) run build
+	@cd $(ICONS_DIR) && $(PNPM) start
```

Comment:
maybe it makes sense to add a dependency installation command here?

Like 

pnmp install && pnpm start

Reply:
UPD: At the moment, we assume that running the command from the root directory should automatically install dependencies in all workspaces.